actors/builtin/storage_power/policy.go

```diff
@@ -34,16 +35,18 @@ func pledgePenaltyForConsensusFault(currPledge abi.TokenAmount, faultType Consen
 }
 
 func rewardForConsensusSlashReport(elapsedEpoch abi.ChainEpoch, collateralToSlash abi.TokenAmount) abi.TokenAmount {
-	autil.TODO()
-	// BigInt Operation
-	// var growthRate = builtin.SLASHER_SHARE_GROWTH_RATE_NUM / builtin.SLASHER_SHARE_GROWTH_RATE_DENOM
-	// var multiplier = growthRate^elapsedEpoch
-	// var slasherProportion = min(INITIAL_SLASHER_SHARE * multiplier, 1.0)
-	// return collateralToSlash * slasherProportion
-	return abi.NewTokenAmount(0)
+	// autil.PARAM_FINISH()
+	growthRate := big.Div(indices.ConsensusFault_SlasherShareGrowthRateNum(), indices.ConsensusFault_SlasherShareGrowthRateDenom())
+	multiplier := big.NewInt(1)
+	for i := int64(0); i < int64(elapsedEpoch); i++ {
+		multiplier = big.Mul(multiplier, growthRate)
+	}
+	initial_slasher_share := big.Div(indices.ConsensusFault_SlasherInitialShareNum(), indices.ConsensusFault_SlasherInitialShareDenom())
+	slasher_proportion := big.Min(big.Mul(initial_slasher_share, multiplier), big.NewInt(1))
+	return big.Mul(collateralToSlash, slasher_proportion)
 }
 
 func consensusPowerForWeight(weight SectorStorageWeightDesc) abi.StoragePower {
-	autil.PARAM_FINISH()
-	panic("")
+	// autil.PARAM_FINISH()
+	return big.Mul(big.NewInt(int64(weight.SectorSize)), big.NewInt(int64(weight.Duration)))
```

Comment:
This looks wrong. Power is measured in bytes, not byte-epochs.

Reply:
Similar to comment at SectorWeight, but will revert to just SectorSize for now